gp-includes/formats/format-json.php

```diff
@@ -106,6 +106,9 @@ public function read_originals_from_file( $file_name ) {
 			}
 
 			$args['translations'] = (array) $value;
+			if ( empty( $args['translations'][0] ) || ! is_string( $args['translations'][0] ) ) {
+				continue;
```

Comment:
```suggestion
				$args['translations'] = array();
```

You cannot `continue` here as this is also used for reading originals without translations. So failing tests. So maybe set it to an empty array? Or can we actually handle the double array since the translations seem to be legit? Why dropping them?

Reply:
An empty array seems like a good solution then. The double array was speculation, what about triple arrays? I think trying to parse invalid JSON will be an uphill battle so I like your solution!